roofit/roofit/src/RooFitComputeInterface.h

```diff
@@ -11,6 +11,13 @@ namespace BatchHelpers {
 }
 
 namespace RooFitCompute {
+  /**
+   * \brief The interface which should be implemented to provide optimised evaluateSpan() functionality for RooFit PDF classes.
+   *
+   * This interface contains the signatures of the compute functions of every PDF that has an optimised implementation available.
+   * These are the functions that perform the actual computations in batches.
+   * \see dispatch, RooFitComputeClass, RF_ARCH
```

Comment:
Does `dispatch` link correctly in doxygen? Maybe you have to use RooFitCompute::dispatch if it doesn't.